### DISCUSSION TITLE
Basic theming

### DISCUSSION THREAD

**Discussion by V3NCO** [dani-garcia/vaultwarden#5056] [Ideas]
Just adding fields in the admin panel to theme the new ui (basically just the sidebar color and the icon) would be a cool feature.

**Comment by BlackDex** [comment]
Probably #4940 will be helpful.
But via the admin interface is not part of that (for now at least).

**Reply by V3NCO** [reply]
Well the admin ui was a suggestion, but this css approach is interesting too
